secant/Features/Send/SendStore.swift

```diff
@@ -37,7 +37,13 @@ struct SendState: Equatable {
     var isSendingTransaction = false
     var totalBalance = 0.0
     var transaction: Transaction
-    var transactionInputState: TransactionInputState
+    var transactionAddressInputState: TransactionAddressInputState
+    var transactionAmountInputState: TransactionAmountInputState
+    
+    var isValidForm: Bool {
```

Comment:
this can seem a silly test to cover but it could lead to avoidable user errors further in the flow

Reply:
tests implemented